test/ir/inference/test_trt_convert_bitwise_not.py

```diff
@@ -70,7 +70,7 @@ def generate_input1(dims, batch, attrs: List[Dict[str, Any]]):
 
     def sample_predictor_configs(
         self, program_config
-    ) -> (paddle_infer.Config, List[int], float):
+    ) -> (paddle_infer.Config, list[int], float):
```

Comment:
```suggestion
    ) -> tuple[paddle_infer.Config, list[int], float]:
```

这里是明显错误的，需要修改

别说一个就改一个，其他的也需要改

Reply:
抱歉，已全部修改